.github/workflows/vale.yml

```diff
@@ -8,21 +8,8 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@master
-    - uses: jwalton/gh-find-current-pr@v1
-      id: findPr
+    - uses: pierrebeitz/vale-action@ce94dd2b4ef576bf1237f45bf2bbd9ed4877e740
```

Comment:
we can replace this with the official action, if/when https://github.com/errata-ai/vale-action/pull/23 merges